<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessBClass</class>
 <widget class="QMainWindow" name="ProcessBClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>读取的值：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addressInput"/>
    </item>
    <item>
     <widget class="QLabel" name="valueLabel">
      <property name="text">
       <string>传输的值：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="valueInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="readButton">
      <property name="text">
       <string>读取</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="writeButton">
      <property name="text">
       <string>写入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
